src/irc/IrcServer.ts

```diff
@@ -80,6 +80,7 @@ export interface IrcServerConfig {
         federate: boolean;
     };
     matrixClients: {
+        matrixIdNormalization: number,
```

Comment:
I thought that I should make this an integer, in case there are more schemes in the future